Assets/_Scripts/ChessPiece.cs

```diff
@@ -48,6 +52,7 @@ public class ChessPiece : MonoBehaviour {
 
     private void _RefreshChessPieceSprite() {
         spriteRenderer.sprite = Services.ChessSprites.GetChessmanSprite(chessman, team);
+        spriteRenderer.color = new Color(Random.Range(0.0f,1.0f), Random.Range(0.0f,1.0f), Random.Range(0.0f,1.0f));

```

Comment:
Ming are you aware this makes all of the chess pieces look like they came from a fruit loops factory and are also throwing up confetti?